external/runtime/runtime.depproj

```diff
@@ -7,18 +7,45 @@
     <NugetRuntimeIdentifier Condition="'$(OSGroup)' == 'Unix' AND '$(RunningOnUnix)' != 'true'">ubuntu.14.04-x64</NugetRuntimeIdentifier>
     <NugetRuntimeIdentifier Condition="'$(TargetGroup)' == 'uap'">win10-$(ArchGroup)</NugetRuntimeIdentifier>
     <NugetRuntimeIdentifier Condition="'$(TargetGroup)' == 'uapaot'">win10-$(ArchGroup)-aot</NugetRuntimeIdentifier>
+    <AppendMSBuildRestoreArguments>true</AppendMSBuildRestoreArguments>
```

Comment:
Oops, I think this is a leftover change from some intermediate state. Will remove.